modules/ROOT/pages/dw-url-types.adoc

```diff
@@ -6,6 +6,6 @@
 | URI
 | `type URI = { isValid: Boolean, host?: String, authority?: String, fragment?: String, path?: String, port?: Number, query?: String, scheme?: String, user?: String, isAbsolute?: Boolean, isOpaque?: Boolean }`
 | Describes the URI type. For descriptions of the fields, see
-https://docs.mulesoft.com/mule-runtime/4.3/dataweave-types#dw_type_url[URL Types (dw::core::URL)].
```

Comment:
TODO: STILL 4.3 in source. should it be latest.

* https://docs.mulesoft.com/mule-runtime/4.3/dataweave-types#dw_type_url[URL Types (dw::core::URL)].